verifier/check_known_issues.py

```diff
@@ -28,125 +33,142 @@
 
 # E.g., NBSP vs SP in NodeJS DateTime in ICU73, ...
 
-class known_issue():
-    # Stores information on a single known issue.
-    # Ths may be stored in file or database
-    def __init__(self):
-        self.description = None
-        self.url_ref = None
-        self.test_type = None
-        self.instances = []
-        self.filter_fn = None
-        self.date_posted = None
-
-        self.issue_list = []
-
 # Constants
 NBSP = '\u202f'
 SP = '\u0020'
 
-# Global KI Info.
-# TODO: add this to a class
-known_issue_nbsp_sp = 'ASCII Space instead of NBSP'
-known_issue_replaced_numerals = 'Not creating non-ASCII numerals'
+
+# Global KnownIssue Info types and strings
+class knownIssueType(Enum):
+    known_issue_nbsp_sp = 'ASCII Space instead of NBSP'
+    known_issue_replaced_numerals = 'Not creating non-ASCII numerals'
 
 # TODO! Load known issues from file of known problems rather than hardcoding
 
-class compute_known_issue():
-    def __init__(self):
-        return
+def compute_known_issues(test_type, test, known_issue_list):
```

Comment:
remove `known_issue_list` from parameters here. instead, append to `known_issues_list` in the caller (`compute_known_issues`)